### ISSUE TITLE
Font weightdifferent when self contained

### ISSUE THREAD

**Issue by vmiwouter** [QuestPDF/QuestPDF#922]
**Describe the bug**
After the last two releases with 2024.3.10 and 2024.6 if i use self contained instead of release/debug i get different thickness and all seems to be bold.

**To Reproduce**
Assign .bold() to random text section.
When publish via self contained everything is bold now.
![image](https://github.com/QuestPDF/QuestPDF/assets/44747792/8427fd56-f8ee-451a-a9cf-0b2821e0f85f)
2024.6 
**Expected behavior**
![image](https://github.com/QuestPDF/QuestPDF/assets/44747792/d7aec259-3d26-4d85-9590-f4bcc8ac37c9)
2024.3.9

**Screenshots**

**Environment**
What version of the library do you use?  2024.6.0
What operating system do you use? Windows Server 2022

**Additional context**

**Comment by vmiwouter** [comment]
> Could you please share your code and/or minimal example? It is not clear what font you are using precisely on the screenshots. The 2024.6 version looks like all text elements use the Anton font. Is that expected?
> 
> Are you using the `Settings.UseEnvironmentFonts` flag set to `false`?
I don't use Settings.UseEnvironmentFonts both are supossed to be same without the Anton font, i did remove traces of in the code.

Its like its related to my visual studio somehow, because it only happens when the self contained is published as single file.
At the example invoice where its based on it doesnt have the issues.

So seems only limited to this project.

**Comment by MarcinZiabek** [comment]
If you find anything during the investigation, please let me know. Not much has change between 2024.3.9 and 2024.3.10 - it was mostly Skia update from m124 to m126 and an enhancement for font support in SVG.

**Comment by vmiwouter** [comment]
I tested a bit more it occurs when i use '<IncludeNativeLibrariesForSelfExtract>true</IncludeNativeLibrariesForSelfExtract>' when publishing the single file.

At a brand new project there are no issues only in the existing one, maybe there is some conflict.

**Comment by vmiwouter** [comment]
> @vmiwouter I was following this issue, were you able to resolve it?

Yes, as workaround i set a default font family and added that to the container :
page.DefaultTextStyle(x => x.FontFamily("Calibri"));

This way i can set different fonts and the external font wont override.

**Comment by luis-fss** [comment]
Cool, thanks for clarifying.
